frontend/packages/console-app/src/components/quick-starts/data/mocks/yamls/quick-start-crd.yaml

```diff
@@ -0,0 +1,201 @@
+apiVersion: apiextensions.k8s.io/v1beta1
```

Comment:
We don't need the CRD yaml anymore. The console operator has been updated with new CRD after this PR merged - https://github.com/openshift/console-operator/pull/480

Reply:
removed the CRD